app/models/job_io_wrapper.rb

```diff
@@ -74,13 +73,13 @@ def file_actor
     Hyrax::Actors::FileActor.new(file_set, relation.to_sym, user)
   end
 
-  # @return [FileNode, FalseClass] the created file node on success, false on failure
+  # @return [Hyrax::FileNode, FalseClass] the created file node on success, false on failure
   def ingest_file
     file_actor.ingest_file(self)
   end
 
   def to_file_node
-    Wings::FileNode.new(label: original_name,
+    Hyrax::FileNode.new(label: original_name,
```

Comment:
There are a number of places using Wings namespace.  It was changed to Hyrax namespace since FileNode was moved there.